src/Boolean.php

```diff
@@ -19,6 +19,14 @@
 use function sprintf;
 use function strtolower;
 
+/**
+ * @psalm-type Options = array{
+ *     type: int-mask-of<self::TYPE_*>,
```

Comment:
Wait, is it a one-of, or an int-mask-of? Confusing across the file 🤔

Reply:
Internally, it's in-mask. As a parameter, it's sometimes a string equivalent

Reply:
Thanks for clarifying 👍